### ISSUE TITLE
Autofill makes clippercard.com unusable

### ISSUE THREAD

**Issue by kentor** [bitwarden/browser#456]
Only after you login, autofill will trigger the search box and submit it, making the site unusable until you turn autofill off.

The username will replace the "Go" button's value and submit it. Probably because this button doesn't have `type="submit"` like it should.

![image](https://user-images.githubusercontent.com/240493/34511352-d92efa64-f00f-11e7-9fc0-e5c80e24448e.png)

**Comment by kspearrin** [comment]
Any idea how I can register an account for free? I guess I need to log in to see this issue?

**Comment by kentor** [comment]
Oh it also happens on this page while logged out: https://www.clippercard.com/ClipperCard/register.jsf

no need to register, just add a login

**Comment by kspearrin** [comment]
Yes, this is happening because the button input does not properly have a type set, so it is treated as a text field. It just so happens to also fall before the hidden password field so it is treated as a username. Not sure what I can do here?

**Comment by kentor** [comment]
I was thinking about an exception list. I ran across https://github.com/bitwarden/browser/issues/317 but I don't mean one that the password manager program maintains, just a list that I can add to for myself.

**Comment by bfranske** [comment]
Related to #501
